### ISSUE TITLE
is it possible to take the enemies of the character graveyerd and place them as enemies to attack? 

### ISSUE THREAD

**Issue by Kratos8790** [mogaika/god_of_war_browser#36]
![INIMIGO2](https://user-images.githubusercontent.com/77935216/106174872-38ae9c00-6174-11eb-87ed-e1e3db278544.PNG)
![INIMIGO3](https://user-images.githubusercontent.com/77935216/106174873-39dfc900-6174-11eb-8058-34cd1f9488d3.PNG)
![inimigo4](https://user-images.githubusercontent.com/77935216/106174876-3a785f80-6174-11eb-9d48-ae7a1e13ce55.PNG)
![inimigo6](https://user-images.githubusercontent.com/77935216/106174879-3a785f80-6174-11eb-8d3e-c2b965becd63.PNG)
![INIMIGO1](https://user-images.githubusercontent.com/77935216/106174891-3cdab980-6174-11eb-8251-5c236038673f.PNG)
![INIMGO5](https://user-images.githubusercontent.com/77935216/106174894-3d735000-6174-11eb-9296-31c6f555fddc.PNG)

**Comment by mogaika** [comment]
No, it's just models in OLYMP04.WAD. They do not have skinning in meshes
